Packs/F5Silverline/Integrations/F5Silverline/F5Silverline.py

```diff
@@ -186,9 +188,11 @@ def delete_ip_objects_command(client: Client, args: Dict[str, Any]):
         human_readable = ""
         for object_id in object_id_list:
             url_suffix = f'{list_type}/ip_objects/{object_id}'
-            client.request_ip_objects(body={}, method='DELETE', url_suffix=url_suffix, params={}, resp_type='content')
+            client.request_ip_objects(body={}, method='DELETE', url_suffix=url_suffix, params={
+                                      'list_target': list_target}, resp_type='content')
             human_readable += f"IP object with ID: {object_id} deleted successfully from the {list_type} list. \n"
         return CommandResults(readable_output=human_readable)
+    return None
```

Comment:
Why return None explicitly?

Reply:
It was a ruff suggestion, its for readability purposes